src/templates/EditorialNewsletter/index.js

```diff
@@ -0,0 +1,260 @@
+import Container from './Container'
+import Cover, { CoverImage } from './Cover'
+import Paragraph, { Strong, Em, Link, Br } from './Paragraph'
+import Center from './Center'
+import { H2 } from './Headlines'
+import Figure, { Image, Caption, Byline } from './Figure'
+import Blockquote, { BlockquoteText, BlockquoteSource } from './Blockquote'
+import List, { ListItem } from './List'
+
+import {
+  matchType,
+  matchZone,
+  matchHeading,
+  matchParagraph,
+  matchImageParagraph
+} from 'mdast-react-render/lib/utils'
+
+import {
+  FigureImage
+} from '../../components/Figure'
+
+import {
+  matchFigure,
+  extractImage,
+  getDisplayWidth
+} from '../Article/utils'
+
+const matchLast = (node, index, parent) => index === parent.children.length - 1
+
+const globalInlines = [
+  {
+    matchMdast: matchType('break'),
+    component: Br,
+    isVoid: true
+  },
+  {
+    matchMdast: matchType('strong'),
+    component: Strong,
+    editorModule: 'mark',
+    editorOptions: {
+      type: 'strong'
+    }
+  }
+]
+
+const link = {
+  matchMdast: matchType('link'),
+  component: Link,
+  props: node => ({
+    title: node.title,
+    href: node.url
+  }),
+  editorModule: 'link'
+}
+
+const paragraph = {
+  matchMdast: matchParagraph,
+  component: Paragraph,
+  editorModule: 'paragraph',
+  editorOptions: {
+    formatButtonText: 'Paragraph'
+  },
+  rules: [
+    ...globalInlines,
+    link,
+    {
+      matchMdast: matchType('emphasis'),
+      component: Em,
+      editorModule: 'mark',
+      editorOptions: {
+        type: 'emphasis'
+      }
+    }
+  ]
+}
+
+const figureCaption = {
+  matchMdast: matchParagraph,
+  component: Caption,
+  editorModule: 'figureCaption',
+  editorOptions: {
+    afterType: 'PARAGRAPH',
+    insertAfterType: 'FIGURE',
+    placeholder: 'Legende'
+  },
+  rules: [
+    {
+      matchMdast: matchType('emphasis'),
+      component: Byline,
+      editorOptions: {
+        placeholder: 'Credit'
+      }
+    },
+    link,
+    ...globalInlines
+  ]
+}
+
+const figure = {
+  matchMdast: matchFigure,
+  component: Figure,
+  editorModule: 'figure',
+  editorOptions: {
+    pixelNote: 'Auflösung: min. 1200x (proportionaler Schnitt)',
+    insertButtonText: 'Bild'
+  },
+  rules: [
+    {
+      matchMdast: matchImageParagraph,
+      // TODO: CoverImage rule trumps this Image rule only in editor, why?
```

Comment:
The reason is slate modules are flat: If you want a different `figureImage` instance pass a `editorOptions.type`. Cover wins because modules are initiated by traversing the tree top down (cover comes before plain figure). Add `editorOptions: {type: 'COVERFIGUREIMAGE'}` below in the cover image rule.